cw_core/lib/node.dart

```diff
@@ -63,6 +63,8 @@ class Node extends HiveObject with Keyable {
   @HiveField(6)
   String? socksProxyAddress;
 
+  bool connectOverTorOnly = false;
```

Comment:
this will not work in many cases (switching wallets, switching nodes for example)
let's find a better way instead of having this variable as a part of the node object, or at least handle other possible cases